### PR TITLE
remove unittest from pre-push hook

### LINKED ISSUE #11081: remove unittest from pre-push hook
i find that the unittest pre-push hook is more annoying than useful. ex:

1. unittests are sometimes broken on master (such as right now) for reasons unrelated to my change, so i end up having to use --no-verify.
2. pre-push hooks aren't run when merging via Github's UI so ultimately there is no guarantee that the unittests are passing
3. unittests take a long time to run and should be checked after-push in Travis anyway

### PR DESCRIPTION
**Pull request by diracdeltas**
fix #11081

### CODE REVIEW

File: tools/lib/ignoredPaths.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 module.exports = [
-  'pre-comit',
+  'pre-commit',
```
Comment (luixxiul) [inline tools/lib/ignoredPaths.js]: I've been not sure if this was a typo: https://github.com/brave/browser-laptop/pull/9690#discussion_r123872847

Comment (diracdeltas) [reply tools/lib/ignoredPaths.js]: cc @NejcZdovc on whether this is a typo

**Comment by diracdeltas** [comment]
@bsclifton that's a reasonable option

**Comment by diracdeltas** [comment]
@bsclifton done!

File: tools/lib/ignoredPaths.js (hunk 1 above)
Comment (bbondy) [reply tools/lib/ignoredPaths.js]: These are sub-dirs of node_modules mostly so yes it's a typo. Thx.

**Pull request merged**